packages/driver/src/cy/listeners.js

```diff
@@ -21,9 +21,16 @@ const removeAllListeners = () => {
 }
 
 const addListener = (win, event, cb) => {
-  events.push([win, event, cb])
+  const id = Math.random()
```

Comment:
This is intended to be a unique ID, right? I prefer to use something like [`_.uniqueId()`](https://lodash.com/docs/4.17.15#uniqueId) or an incrementing number in that case. The possibility of a collision with a random number is pretty unlikely, but not impossible. The problem is if a collision does occur, it can surface in an obscure way and be very difficult to debug since it's literally random. Using a unique ID won't have that problem and also expresses the intent more clearly.

Reply:
This is gone in the new implementation